kivy/properties.pyx

```diff
@@ -1983,11 +1983,18 @@ cdef class ConfigParserProperty(Property):
 cdef class ColorProperty(Property):
     '''Property that represents a color. The assignment can take either:
 
-    - a list or tuple of 3 to 4 float values between 0-1 (kivy default)
+    - a collection of 3 or 4 float values between 0-1 (kivy default)
     - a string in the format #rrggbb or #rrggbbaa
+    - a string representing color name (eg. 'red', 'yellow', 'green')
+
+    Color names definitions can be found at this
+    `link <https://www.w3.org/TR/SVG11/types.html#ColorKeywords>`_. Color can
```

Comment:
It'd be nice to also link to `kivy.utils.colormap` from here.

Reply:
I'll add:

"...found at his `link` and at :obj:`kivy.utils.colormap`."